Classes/Utility/ConfigUtility.php

```diff
@@ -221,4 +221,15 @@ public function getGlobalMailFrom(): string
     {
         return $this->getExtConf('from_mail', $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailFromAddress'] ?? '');
     }
+
+    public function getExcludedLogMessageRegularExpressions(): array
+    {
+        $regularExpressions = $this->getExtConf('excluded_log_message_regular_expressions', []);
+
+        if (!is_array($regularExpressions)) {
+            return GeneralUtility::trimExplode(',', $regularExpressions);
```

Comment:
You realy want to explode the regex by a comma? you cant use a comma in regex this way